src/components/Card.tsx

```diff
@@ -142,79 +142,18 @@ const StepCardHeader = styled.div`
 `;
 StepCardHeader.displayName = 'StepCardHeader';
 
-export const StepCardFooter = styled.div`
-    padding: var(--step-card-padding-top) var(--step-card-padding-side);
-    border-top: 1px solid ${colors.palette.pale};
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: space-between;
-    font-size: 1.4rem;
-    line-height: 2rem;
-
-    > * {
-        flex-grow: 1;
-    }
-
-    button {
-        width: 160px;
-        height: 48px;
-    }
-
-    .points {
-        margin-bottom: 1.6rem; // Replace with https://caniuse.com/?search=gap soon
-
-        .attempts-left {
-            color: #F36B32;
-        }
-    }
-
-    .controls {
-        display: flex;
-        flex-flow: column wrap-reverse;
-        margin-left: 1.6rem; // Replace with https://caniuse.com/?search=gap soon
-
-        button + button {
-            margin: 0.8rem 0 0 0;
-        }
-    }
-
-    ${breakpoints.desktop`
-        padding: 32px var(--step-card-padding-side);
-        flex-wrap: nowrap;
-
-        .points {
-            max-width: 400px;
-        }
-
-        .controls {
-            flex-flow: row;
-            justify-content: flex-end;
-
-            button + button {
-                margin: 0 0 0 0.8rem;
-            }
-        }
-    `}
-`;
-
 const StepCardQuestion = styled.div<{ unpadded?: boolean }>`
-    --step-card-padding-top: 48px;
-    --step-card-padding-side: 140px;
+  .step-card-body {
+    padding: 48px 140px;
 
-    ${breakpoints.only.tablet`
-        --step-card-padding-top: ${breakpoints.margins.tablet};
-        --step-card-padding-side: ${breakpoints.margins.tablet};
+    ${breakpoints.tablet`
+      padding: ${breakpoints.margins.tablet} ${breakpoints.margins.tablet};
     `}
 
-    ${breakpoints.only.mobile`
-        --step-card-padding-top: calc(${breakpoints.margins.mobile} * 2);
-        --step-card-padding-side: ${breakpoints.margins.mobile};
+    ${breakpoints.mobile`
+      padding: calc(${breakpoints.margins.mobile} * 2) ${breakpoints.margins.mobile};
     `}
-
-
-    ${props => props.unpadded ? '.step-card-body' : '&'} {
-        padding: var(--step-card-padding-top) var(--step-card-padding-side);
-    }
```

Comment:
I propose abandoning this rule for now as it complicates managing padding for child components and I didn't see any StepCards in tutor with it set to false. But let me know if you think we should try to make it work here.

Reply:
Agree, I think it was only true in one niche render case for Tutor if even that